src/test/struct.c

```diff
@@ -175,7 +178,17 @@ main( hypre_int argc,
    HYPRE_MatvecFunctions matvec_fn;
    /* end lobpcg */
 
+   /* default execution policy and memory space */
+#if defined(HYPRE_TEST_USING_HOST)
```

Comment:
@liruipeng Are you planning to implement in this PR the config file for setting additional driver options?